.github/workflows/tests.yaml

```diff
@@ -54,14 +54,6 @@ jobs:
         if: matrix.dependencies != 'locked'
         run: composer config --unset platform.php
 
-      - name: Configure for PHP >= 8.3
-        if: "matrix.php-version >= '8.3'"
-        run: composer config platform.php 8.2.99
-
-      - name: Configure minimum-stability
```

Comment:
maybe we should instead configure it for the future version (PHP 8.4 and sf 8.0)?

Reply:
We could revert this change once we introduce support of new versions IMO

Reply:
If we remember it :P

Reply:
I did it a couple of times already ;)